### DISCUSSION TITLE
Shopping List Keep-Awake

### DISCUSSION THREAD

**Discussion by Grygon** [mealie-recipes/mealie#2455] [Feature Request]
**Before submitting this feature request I have**
_COMPLETE AND FILL THESE OUT_
- [x] Reviewed the documentation and verified that my feature request isn't already a feature.
- [x] Reviewed the existing feature requests and my feature has not been requested.
- [x] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**
Currently, the Shopping List doesn't take advantage of the same keep-awake functionality we have on the Recipe Pages. It'd be nice if it did, sometimes it takes a little while to find stuff in my lists.

**Additional Information**
- [x] If the feature is accepted I'm willing to submit a PR to provide this feature

**Comment by michael-genson** [comment]
Huh, didn't even think about that. I think I just subconsciously lock my phone every time I pull it up. Makes sense to me!
